problem_builder/instructor_tool.py

```diff
@@ -295,12 +296,17 @@ def start_export(self, data, suffix=''):
         user_service = self.runtime.service(self, 'user')
         if not self.user_is_staff():
             return {'error': 'permission denied'}
-        if not username:
-            user_id = None
+        if not usernames:
+            user_ids = None
         else:
-            user_id = user_service.get_anonymous_user_id(username, unicode(self.runtime.course_id))
-            if user_id is None:
-                self.raise_error(404, _("Could not find the specified username."))
+            user_ids = []
+            for username in re.split(r',', usernames):
```

Comment:
I don't think you need `re.split` here - can't you just use `usernames.split(',')` ?